### PR TITLE
Handle InputException message

### PR DESCRIPTION
**Pull request by yogeshsuhagiya**
### Description (*)
- Handle the InputException message like:
1. The password needs at least 8 characters. Create a new password and try again.
2. Minimum of different classes of characters in password is 3. Classes of characters: Lower Case, Upper Case, Digits, Special Characters.
3. A customer with the same email address already exists in an associated website.

etc.

### Fixed Issues (if relevant)
1. magento/graphql-ce#446: Internal server error for mutation createCustomer

### Manual testing scenarios (*)
**Perform query:**
```
mutation {
  createCustomer(
    input: {
    	firstname: "Test"
    	lastname: "Customer"
    	email: "test_customer@gmail.com"
    	password: "123456"
    }
  ) {
    customer {
      id
    }
  }
}
```

### Contribution checklist (*)
 - [ ] Pull request has a meaningful description of its purpose
 - [ ] All commits are accompanied by meaningful commit messages
 - [ ] All new or changed code is covered with unit/integration tests (if applicable)
 - [ ] All automated tests passed successfully (all builds on Travis CI are green)

### CODE REVIEW

File: app/code/Magento/CustomerGraphQl/Model/Resolver/CreateCustomer.php [hunk 1]
```diff
@@ -88,6 +89,8 @@ public function resolve(
             throw new GraphQlInputException(__($e->getMessage()));
         } catch (InputMismatchException $e) {
             throw new GraphQlInputException(__($e->getMessage()));
+        } catch (InputException $e) {
```
Comment (naydav) [inline app/code/Magento/CustomerGraphQl/Model/Resolver/CreateCustomer.php]: Looks like we can process just `LocalizedException`
Also, do'nr forget to pass previous exception as parameter of new exception
`throw new GraphQlInputException(__($e->getMessage()), $e);`
Thanks

**Review by naydav — CHANGES_REQUESTED** [review]

File: app/code/Magento/CustomerGraphQl/Model/Resolver/CreateCustomer.php (hunk 1 above)
Comment (yogeshsuhagiya) [reply app/code/Magento/CustomerGraphQl/Model/Resolver/CreateCustomer.php]: Hi @naydav, thanks for the suggestion. 
Modified `CreateCustomer.php` file as per your instruction, please take a look.

**Comment by yogeshsuhagiya** [comment]
Hi @naydav , please review this PR and let me know further modification if requires.

**Comment by yogeshsuhagiya** [comment]
@naydav Can you please review this PR? and let me know if anything requires from my side.

**Comment by naydav** [comment]
> @naydav Can you please review this PR? and let me know if anything requires from my side.

@yogeshsuhagiya Could you merge last mainline and check again? Looks like it's was fixed in PR with big refactoring
https://github.com/magento/graphql-ce/pull/459

Thanks

**Pull request closed without merging**